nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -114,17 +115,50 @@ def check_doi_publications(nwbfile: NWBFile):
 
 @register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=Subject)
 def check_subject_age(subject: Subject):
-    """Check if the Subject age is in ISO 8601."""
+    """Check if the Subject age is in ISO 8601 or our extension of it for ranges."""
     if subject.age is None:
         if subject.date_of_birth is None:
             return InspectorMessage(message="Subject is missing age and date_of_birth.")
-    elif not re.fullmatch(duration_regex, subject.age):
-        return InspectorMessage(
-            message=(
-                f"Subject age, '{subject.age}', does not follow ISO 8601 duration format, e.g. 'P2Y' for 2 years "
-                "or 'P23W' for 23 weeks."
-            )
+        else:
+            return
+    if re.fullmatch(pattern=duration_regex, string=subject.age):
+        return
+
+    if "/" in subject.age:
+        subject_lower_age_bound, subject_upper_age_bound = subject.age.split("/")
+
+        if re.fullmatch(pattern=duration_regex, string=subject_lower_age_bound) and (
+            re.fullmatch(pattern=duration_regex, string=subject_upper_age_bound) or subject_upper_age_bound == ""
+        ):
+            return
+
+    return InspectorMessage(
+        message=(
+            f"Subject age, '{subject.age}', does not follow ISO 8601 duration format, e.g. 'P2Y' for 2 years "
+            "or 'P23W' for 23 weeks. You may also specify a range using a '/' separator, e.g., 'P1D/P3D' for an "
+            "age range somewhere from 1 to 3 days. If you cannot specify the upper bound of the range, "
+            "you may leave the right side blank, e.g., 'P90Y/' means 90 years old or older."
         )
+    )
+
+
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=Subject)
+def check_subject_proper_age_range(subject: Subject):
+    """Check if the Subject age, if specified as duration range (e.g., 'P1D/P3D'), has properly increasing bounds."""
+    if subject.age is not None and "/" in subject.age:
+        subject_lower_age_bound, subject_upper_age_bound = subject.age.split("/")
+
+        if (
+            re.fullmatch(pattern=duration_regex, string=subject_lower_age_bound)
+            and re.fullmatch(pattern=duration_regex, string=subject_upper_age_bound)
```

Comment:
```suggestion
            subject_lower_age_bound and subject_upper_age_bound
```

I guess you are trying to determine if they are durations or empty strings. This seems simpler to me

Reply:
I'm assuring that calling `Timedelta` on them in the line below (https://github.com/NeurodataWithoutBorders/nwbinspector/pull/251/files#diff-865fe408ddf22ed8f087905d06d3130e182b5a4a19f988a01a02563c54b09912R154) does not cause an error in parsing by ensuring each is proper ISO duration

Reply:
ah ok

Reply:
Also lines 130 and 131 are a different function

This could also be taken care of here as an early return

```python
if not re.fullmatch(pattern=duration_regex, string=subject_lower_age_bound) or not re.fullmatch(pattern=duration_regex, string=subject_upper_age_bound):
    return
```